tools/eksDistroBuildToolingOpsTools/docker/Dockerfile

```diff
@@ -9,4 +9,6 @@ ADD $BINARY_OUTPUT_PATH /usr/bin
 
 RUN chmod +x /usr/bin/$BINARY_NAME
 
+RUN yum install git -y
```

Comment:
As a matter of best practice, when yum installing in dockerfiles, clean up afterwards...

https://github.com/aws/eks-distro-build-tooling/blob/main/eks-distro-base/Dockerfile.base#L26-L27

Reply:
Id also use `git-core` instead of `git`. git includes a number of extra deps that you most likely dont need in a container image like this